code/ship/ship.cpp

```diff
@@ -7362,6 +7362,7 @@ void ship_cleanup(int shipnum, int cleanup_mode)
 
 	ship *shipp = &Ships[shipnum];
 	object *objp = &Objects[shipp->objnum];
+	char *jumpnode_name = nullptr;
```

Comment:
Maybe this declaration could be moved a bit closer to the usages.

Reply:
I considered that, but I put it near the beginning of the function because the other declarations were there too.  Plus having it there keeps the code flow in the middle of the function clean.